src/lambda.js

```diff
@@ -43,13 +46,20 @@ const deliverEmail = (event, context, callback) => {
   }
 };
 
-const generateEmail = (event, context, callback) => {
+const generateEmail = async (event, context, callback) => {
```

Comment:
need to add try/catch blocks to this function

Reply:
done